train.py

```diff
@@ -0,0 +1,164 @@
+import uuid
+import itertools
+from typing import Callable
+
+import hydra
+import torch.optim
+import torch.nn as nn
+from tqdm import tqdm
+from omegaconf import DictConfig
+from torch.utils.data import DataLoader
+
+import wandb
+from model import PitchSeqNN
+from data.dataset import BagOfPitches
+
+
+@hydra.main(version_base=None, config_path="conf", config_name="config")
+def main(cfg: DictConfig):
+    # loading data
+    dataset = BagOfPitches()
+    v_dataset = BagOfPitches(split="validation")
+    train_dataloader = DataLoader(dataset, batch_size=cfg.hyperparameters.batch_size, shuffle=True)
+    v_dataloader = DataLoader(v_dataset, batch_size=cfg.hyperparameters.batch_size)
+
+    # get composers to classify against each other
+    count = dataset.df.groupby(["composer"]).size()
+    composers_with_most_data = count[count.values > 10]
+    composers_to_check = composers_with_most_data.index.tolist()
+    composers = [pair for pair in itertools.combinations(composers_to_check, r=2)]
+    print(f"pairs to check: {len(composers)}")
+
+    # container for trained models
+    models = []
+
+    # train model for each pair
+    for pair in composers:
+        print(f"{pair[0]} vs {pair[1]}")
+        model = run_experiment(cfg=cfg, train_data=train_dataloader, val_data=v_dataloader, classnames=pair)
+        models.append((model, pair))
+        path = f"models/{pair[0].replace(' ', '_').lower()}-{pair[1].replace(' ', '_').lower()}.pth"
```

Comment:
```suggestion
        first_composer = pair[0].replace(' ', '_').lower()
        other_composer = pair[1].replace(' ', '_').lower()
        path = f"models/{first_composer}-{other_composer}.pth"
```

I think it's best to avoid writing code inside strings :)